src/app/static/src/app/components/forms/types.ts

```diff
@@ -47,7 +57,7 @@ export const formMeta: DynamicFormMeta = {
                         name: "area_scope",
                         type: "multiselect",
                         options: [{id: "MWI", label: "Malawi"}, {id: "MWI.1", label: "Central"}],
-                        default: "MWI",
```

Comment:
Have replaced "default" with "value", which is always set after the form is rendered, but can optionally be passed in the original metadata if a default is wanted.